### PR TITLE
Update DG to include find command

### PR DESCRIPTION
**Pull request by limzhenwy**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -276,12 +284,14 @@ FApro seeks to improve the quality of life of financial advisors (FAs). It allow
...
 | `* * *`  | user                                       | find a person by name                                                        | locate details of persons without having to go through the entire list |
+| `* * *`  | financial advisor                          | find a person by address                                                     | line-up all my client meetings efficiently                             |
 | `* * *`  | financial advisor                          | edit contact details of clients                                              | client details are up to date                                          |
-| `* *`    | impatient financial advisor                | be able to use and understand the functionalities easily through a help page | I do not have to waste too much time learning how to operate the app   |
+| `* * *`  | financial advisor                          | edit contact details of clients                                              | client details are up to date                                          |
```
Comment (Daphne789) [inline docs/DeveloperGuide.md:337]: I think this user story has been mentioned previously in line 292.

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -408,7 +418,48 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
         Use case resumes at step 2.
 
-    
+**Use case: Find a person**
```
Comment (Daphne789) [inline docs/DeveloperGuide.md]: There are still trails of the use case for the Find feature from the initial version of the DG in line 357 until 384, which need to be removed.

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -238,6 +238,14 @@ _{more aspects and alternatives to be added}_
 
 _{Explain here how the data archiving feature will be implemented}_
 
+### Find by address feature
+
+#### Implementation
+
+The find by address/find_add feature filters the list of contacts in the address book based on their address.
+
+Given below is an example usage scenario and how the find_add feature works.
```
Comment (Daphne789) [inline docs/DeveloperGuide.md:281]: I want to ask what this statement refers to. Does this refer to the User Story and the Use Case?

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (limzhenwy) [reply docs/DeveloperGuide.md]: Completely missed that, thanks for bringing it up!

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (limzhenwy) [reply docs/DeveloperGuide.md:337]: Fixed!

File: docs/DeveloperGuide.md (hunk 3 above)
Comment (limzhenwy) [reply docs/DeveloperGuide.md:281]: LGTM

**Comment by bhnuka** [comment]
LGTM!

**Pull request merged**